dcbr-api/api/models.py

```diff
@@ -1,40 +1,261 @@
+
+    class Meta:
+        verbose_name_plural = "RiskFactorsMeta"
+
+class Risk_Factor_Data(models.Model):
+
+    num_dogs_intact = models.IntegerField(default=0)
+    num_litter_whelped = models.IntegerField(default=0)
+    num_cats_intact = models.IntegerField(default=0)
+    num_litter_queened = models.IntegerField(default=0)
+
+    num_sold = models.IntegerField(default=0)
+    num_transferred = models.IntegerField(default=0)
+    num_traded = models.IntegerField(default=0)
+    num_leased = models.IntegerField(default=0)
+
+    created = models.DateTimeField(auto_now_add=True)
+    updated = models.DateTimeField(auto_now=True)
+
+    operator = models.ForeignKey(
+        Operator, on_delete=models.CASCADE, related_name="risk_data"
+    )
+
+    def __str__(self):
+        return self.operator_type
+
+    def save(self, *args, **kwargs):
+        # self.operator = self.operator
+        super().save(*args, **kwargs)
+
+    class Meta:
+        verbose_name_plural = "RiskFactorsData"
+
+
+
+class Association_Membership(models.Model):
+
+    assocName = models.CharField(max_length=50, default="", blank=True)
+    membershipNum = models.CharField(max_length=10, default="", blank=True)
+    assoc_URL = models.TextField(validators=[URLValidator()])
+    created = models.DateTimeField(auto_now_add=True)
+    updated = models.DateTimeField(auto_now=True)
+
+    operator = models.ForeignKey(
+        Operator, on_delete=models.CASCADE, related_name="associations"
+    )
+
+    def __str__(self):
+        return self.assocName
+
+    def save(self, *args, **kwargs):
+        # self.operator = self.operator
+        super().save(*args, **kwargs)
+
+    class Meta:
+        verbose_name_plural = "Associations"
+
+
+class Inspection(models.Model):
+    regNum = models.CharField(max_length=14)
+    op_firstName = models.CharField(max_length=32)
+    op_middleName = models.CharField(max_length=50)
+    op_lastName = models.CharField(max_length=50)
+
+    soc_1 = models.BooleanField(default=False)
+    soc_2 = models.BooleanField(default=False)
+    soc_3 = models.BooleanField(default=False)
+    soc_4 = models.BooleanField(default=False)
+    soc_5 = models.BooleanField(default=False)
+    soc_6 = models.BooleanField(default=False)
+    soc_7 = models.BooleanField(default=False)
+    soc_8 = models.BooleanField(default=False)
+    soc_9 = models.BooleanField(default=False)
+    soc_10 = models.BooleanField(default=False)
+    soc_11 = models.BooleanField(default=False)
+
+    soc_1_comment = models.TextField()
+    soc_2_comment = models.TextField()
+    soc_3_comment = models.TextField()
+    soc_4_comment = models.TextField()
+    soc_5_comment = models.TextField()
+    soc_6_comment = models.TextField()
+    soc_7_comment = models.TextField()
+    soc_8_comment = models.TextField()
+    soc_9_comment = models.TextField()
+    soc_10_comment = models.TextField()
+    soc_11_comment = models.TextField()
+    soc_12_comment = models.TextField()
```

Comment:
We have two more comment fields compred to the SoC boolean fields: is that correct?